sections/01_abstract.md

```diff
@@ -3,3 +3,14 @@
 Offspring of the [Deep Review](https://github.com/greenelab/deep-review).
 We cite the Deep Review like [@doi:10.1101/142760] or [@tag:deep_review].
 The source repository is [@url:https://github.com/greenelab/manubot-rootstock].
+
+Equations can be integrated and labeled like this `{{ '{{ \'{#eq:label}\' }}' }}` and then
+referenced using `{@eq:label}` like this for Equation @eq:line.
+
+$$ y = mx + b $$ {{ '{#eq:line}' }}
+
+Images can be included and labeled like `{{ '{{ \'{#fig:label}\' }}' }}` and then
```

Comment:
Wow a bunch of stuff I wrote here didn't save... bummer. Basically, I think we can find a solution with jinja.

Does `{ #fig:label}` not work... wouldn't that avoid the opening [comment](http://jinja.pocoo.org/docs/2.9/templates/#comments)? Could we override comment_start_string in our [jinja2 environment](http://jinja.pocoo.org/docs/2.9/api/#jinja2.Environment)? For example, `comment_start_string='{##'`?